src/lib/logical_query_plan/static_table_node.hpp

```diff
@@ -21,8 +21,9 @@ class StaticTableNode : public EnableMakeForLQPNode<StaticTableNode>, public Abs
   std::vector<std::shared_ptr<AbstractExpression>> output_expressions() const override;
   bool is_column_nullable(const ColumnID column_id) const override;
 
-  // Generates unique constraints from table's key constraints.
-  std::shared_ptr<LQPUniqueConstraints> unique_constraints() const override;
+  // Generates UCCs from table's key constraints. We use these UCCs for storing table key constraints with CREATE TABLE
+  // statements.
```

Comment:
The new comment is confusing to me: These UCCs, generated by this function, are used for storing table key constraints with CREATE TABLE statement? Aren't they stored as soft key constraints?

```suggestion
  // Generates UCCs from table's soft key constraints. We use these soft key constraints for storing table key constraints with CREATE TABLE
  // statements.
```

Reply:
You're right